src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -246,14 +258,17 @@ public UserResponse<T> AskUserToChooseOrCreateNew<T>(IEnumerable<T> options, str
             };
         }
 
-        public string AskUserForValue(string message, string defaultValue, bool allowEmpty, string resetValue = "", params Func<string, string>[] validators)
+        public string AskUserForValue(string message, string defaultValue, bool allowEmpty, string resetValue = "", bool selectOrCreateNewStackName = false, params Func<string, string>[] validators)

```

Comment:
I don't want to have to add another bool every time we come up with another question we need a special label. We should just allow the method to take an option "Enter" label. If one exists this function uses otherwise uses the default.

Reply:
Replaced the bool with flexible default prompts and default labels.